.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: go
 go: go1.9
 install:
+  - go get -t ./...
```

Comment:
I think the new dep needs to be added to glide.yaml
maybe we should move this project to go.mod instead.

Reply:
Done.